config.json

```diff
@@ -23,11 +23,10 @@
         ]
     },
     "redis": {
-      "host": "127.0.0.1",
-      "port": 6380,
-      "db": 0,
-      "user": "",
-      "password": "vgeuwrKZLYwNyZXNc7vTFbjJ4vQnMGrs"
+        "host": "127.0.0.1",
+        "port": 6380,
+        "db": 0,
+        "password": null
```

Comment:
went through the docs again, user field doesn't mean anything for redis.
Empty string is a valid password in redis. System can have redis configuration which has `requirepass ""`. So, we need to differentiate b/w two.